docs/README.md

```diff
@@ -8,18 +8,21 @@
     </picture>
 
 </p>
-<p align="center">
+<p align="left">
```

Comment:
The image shields are now left-aligned while the header and architecture images are center-aligned.